package.json

```
@@ -92,13 +92,14 @@
     "build-js": "npx webpack",
     "build-js:watch": "npx webpack --mode=development --watch",
     "build-demo-module": "npx esbuild src/BookReaderTemplate/BookReaderTemplate.js --outfile=BookReaderDemo/bookreader-template-bundle.js --bundle",
+    "build-demo-watch": "npx esbuild src/BookReaderTemplate/BookReaderTemplate.js --outfile=BookReaderDemo/bookreader-template-bundle.js --bundle --watch",
```

Consistency with other ones here

```suggestion
    "build-es": "npx esbuild src/BookReaderTemplate/BookReaderTemplate.js --outfile=BookReaderDemo/bookreader-template-bundle.js --bundle",
    "build-es:watch": "npx esbuild src/BookReaderTemplate/BookReaderTemplate.js --outfile=BookReaderDemo/bookreader-template-bundle.js --bundle --watch",
```